core/steps/microvm/start_test.go

```diff
@@ -44,25 +44,32 @@ func TestNewStartStep(t *testing.T) {
 	ctx := context.Background()
 	vm := testVMToStart()
 
-	step := microvm.NewStartStep(vm, microVMService)
+	step := microvm.NewStartStep(vm, microVMService, 1)
```

Comment:
can we var these `1`s here? just so it is easy to know at a glance what this param is meant to be